epistles-v1/original/14-epistles.adoc

```diff
@@ -460,8 +461,13 @@ was before death and impatient spirits were.
 
 === Concerning the worship of God, and the Jew outward, and the inward Jew`'s offering, with their garments.
```

Comment:
Can you make this a `[.blurb]` -- and also the two other `=== Concerning...` headlines in this epistle?

Reply:
I'm sorry about the headings, I misread Jason's message I just realized, and thought he said you did most of the headings and stuff so I didn't really bother with them thinking I would be undoing your work. 😬

Reply:
No need to apologize, you were right - I did basically do all of the headings, I just didn't notice these, so it was my oversight really. :smile: